react-app/src/library/types/call-info.ts

```diff
@@ -0,0 +1,9 @@
+export default class CallInfo {
+    conversationId: string;
+    contactName: string;
+
+    constructor(conversationId: string, contactName: string) {
+        this.conversationId = conversationId;
+        this.contactName = contactName;
+    }
+}
```

Comment:
Does this need to be a class; can it just be a type? I don't see any new instantiations anyway except the tests.